exercises/change/cases_test.go

```diff
@@ -0,0 +1,138 @@
+package change
+
+// Source: exercism/x-common
+// Commit: cda8f98 Create new exercises structure
+
+var testCases = []struct {
+	description string
+	coins       []int
+	target      int
+	expected    []int
+}{
+	{
+		"single coin change",
+		[]int{
+			1,
+			5,
+			10,
+			25,
+			100,
+		},
+		25,
+		[]int{
+			25,
+		},
+	},
+	{
+		"multiple coin change",
+		[]int{
+			1,
+			5,
+			10,
+			25,
+			100,
+		},
+		15,
+		[]int{
+			5,
+			10,
+		},
+	},
+	{
+		"change with Lilliputian Coins",
+		[]int{
+			1,
+			4,
+			15,
+			20,
+			50,
+		},
+		23,
+		[]int{
+			4,
+			4,
+			15,
+		},
+	},
+	{
+		"change with Lower Elbonia Coins",
+		[]int{
+			1,
+			5,
+			10,
+			21,
+			25,
+		},
+		63,
+		[]int{
+			21,
+			21,
+			21,
+		},
+	},
+	{
+		"large target values",
+		[]int{
+			1,
+			2,
+			5,
+			10,
+			20,
+			50,
+			100,
+		},
+		999,
+		[]int{
+			2,
+			2,
+			5,
+			20,
+			20,
+			50,
+			100,
+			100,
+			100,
+			100,
+			100,
+			100,
+			100,
+			100,
+			100,
+		},
+	},
+	{
+		"no coins make 0 change",
+		[]int{
+			1,
+			5,
+			10,
+			21,
+			25,
+		},
+		0,
+		[]int{},
+	},
+	{
+		"error testing for change smaller than the smallest of coins",
+		[]int{
+			5,
+			10,
+		},
+		3,
+		[]int{
+			-1,
```

Comment:
even though the JSON file https://github.com/exercism/x-common/blob/master/exercises/change/canonical-data.json#L66 uses -1 for this, I might not put it in a slice `[]int{-1}`, since in the JSON file it's just the single integer -1 instead of `[-1]`. Perhaps the `Change` function could return `([]int, error)` or `([]int, bool)` to make it clear when change can't be made, instead of using this sentinel value